tests/qgis-projects/tests/test_filter_layer_data_by_polygon_for_groups/lizmap_polygon_filter.py

```diff
@@ -0,0 +1,127 @@
+# Headers and parameters
```

Comment:
This file can be removed, integrated in https://github.com/3liz/lizmap-plugin/pull/355